velox/docs/develop/testing/fuzzer.rst

```diff
@@ -178,14 +178,35 @@ aggregate functions supported by the engine, and call
 ``AggregationFuzzerRunner::run()`` defined in `AggregationFuzzerRunner.h`_. See
 `AggregationFuzzerTest.cpp`_.
 
-.. _AggregationFuzzerRunner.h: https://github.com/facebookincubator/velox/blob/main/velox/exec/tests/AggregationFuzzer.h
+.. _AggregationFuzzerRunner.h: https://github.com/facebookincubator/velox/blob/main/velox/exec/fuzzer/AggregationFuzzer.h
 
-.. _AggregationFuzzerTest.cpp: https://github.com/facebookincubator/velox/blob/main/velox/exec/tests/AggregationFuzzerTest.cpp
+.. _AggregationFuzzerTest.cpp: https://github.com/facebookincubator/velox/blob/main/velox/functions/prestosql/fuzzer/AggregationFuzzerTest.cpp
 
 Aggregation Fuzzer allows to indicate functions whose results depend on the
 order of inputs and optionally provide custom result verifiers. The Fuzzer
 also allows to provide custom input generators for individual functions.
 
+Integration with the Window Fuzzer is similar to Aggregation Fuzzer. See
+`WindowFuzzerRunner.h`_ and `WindowFuzzerTest.cpp`_.
+
+`WindowFuzzerTest.cpp`_ allows results to be verified against Presto. To setup
+Presto as a reference DB, please follow these `instructions`_. Once the Presto
+cluster is running, the url should be passed in the argument ``--presto_url``,
+and the request timeout (in milliseconds) can be passed in the argument
+``--req_timeout_ms``. During testing, a higher timeout should be used if certain
+queries are reported to have failed in the reference DB (Presto) but they are
+not submitted to Presto. Example command:
+
+::
+
+    velox/functions/prestosql/fuzzer:velox_window_fuzzer_test --enable_window_reference_verification --presto_url="http://127.0.0.1:8080" --req_timeout_ms=2000 --duration_sec=60 --logtostderr=1 --minloglevel=0
```

Comment:
I actually think the better place to add this is in the How to run section below or create a new section on using Presto as a source of truth ?

Reply:
Moved it to `How to run` section, thanks!